business/badgeCalculator.js

```diff
@@ -12,23 +12,14 @@ const scoreToUrl = {
   1: 'https://img.shields.io/badge/ClearlyDefined-1-yellow.svg',
   2: 'https://img.shields.io/badge/ClearlyDefined-2-brightgreen.svg'
 }
+
```

Comment:
Can this be removed completely now? the new browser code just computes this. the only reason we were doing it here was to enable the browser to ask the service. That is no longer the case.

Note that also means we can remove the tests and the route